app/code/Magento/Customer/Model/Checkout/ConfigProvider.php

```diff
@@ -23,24 +23,24 @@ class ConfigProvider implements ConfigProviderInterface
     /**
      * @var UrlInterface
      */
-    protected $urlBuilder;
+    protected $customerUrl;
 
     /**
      * @var ScopeConfigInterface
      */
     protected $scopeConfig;
 
     /**
-     * @param UrlInterface $urlBuilder
+     * @param Url $customerUrl
      * @param StoreManagerInterface $storeManager
      * @param ScopeConfigInterface $scopeConfig
      */
     public function __construct(
-        UrlInterface $urlBuilder,
```

Comment:
According to [Backward compatible development guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter), you can't modify existing parameters. Add `Url` like optional parameter.